docs/testrunner-toolkit/integrations/github-actions.md

```diff
@@ -60,42 +60,36 @@ jobs:
 
 ## Create the Test Job
 
-In the examples below, the environment variables (`env`) equate to the values configured in GitHub secrets (see above steps). These events only trigger test runs `on` every `pull_request` and/or `push` to the `master` branch.
+In the examples below, we illustrate the different run modes that `saucectl` has: Docker and the Sauce Cloud. They determine where the tests are executed. Docker means that tests are executed locally in a container, while Sauce refers to the Sauce Cloud (i.e. Sauce Labs infrastructure).
+If you are running your tests on the Sauce Cloud, you will likely require a tunnel back to where your application is running. A tunnel will enable the remote browser to access your local network.
+For this, we are going to use [Sauce Connect](secure-connections/sauce-connect.md).
 
 > For more detailed information on setting event-driven actions and jobs, please visit the [GitHub Action documentation](https://docs.github.com/en/free-pro-team@latest/actions/learn-github-actions/introduction-to-github-actions#the-components-of-github-actions).
 
 <Tabs
-  defaultValue="cypress"
+  defaultValue="Docker"
   values={[
-    {label: 'Cypress', value: 'cypress'},
-    {label: 'Playwright', value: 'playwright'},
-    {label: 'TestCafe', value: 'testcafe'},
+    {label: 'Docker', value: 'Docker'},
+    {label: 'Sauce Cloud', value: 'Sauce Cloud'},
   ]}>
   
-<TabItem value="cypress">
+<TabItem value="Docker">
 
 ```yaml reference
 https://github.com/saucelabs/testrunner-toolkit/blob/master/.github/workflows/tests.yml#L93-L114
 ```
 
 </TabItem>
-<TabItem value="playwright">
+<TabItem value="Sauce Cloud">
 
 ```yaml reference
-https://github.com/saucelabs/testrunner-toolkit/blob/master/.github/workflows/tests.yml#L44-L67
-```
-
-</TabItem>
-<TabItem value="testcafe">
-
-```yaml reference
-https://github.com/saucelabs/testrunner-toolkit/blob/master/.github/workflows/tests.yml#L69-L91
+https://github.com/saucelabs/sauce-docs/blob/master/.github/workflows/deploy.yml#L79-L93
 ```
 
 </TabItem>
 </Tabs>
 
-> You can reference our example workflows [here](https://github.com/saucelabs/testrunner-toolkit/tree/master/.github/workflows).
+> You can reference our example workflows [here](https://github.com/saucelabs/testrunner-toolkit/tree/master/.github/workflows) and [here](https://github.com/saucelabs/sauce-docs/blob/master/.github/workflows).
```

Comment:
I think the GH plugin already provides a "see the full example on GitHub" at the bottom of the code box.